packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts

```diff
@@ -30,10 +30,10 @@ export function updateTsConfig(targetPath: string, projectName: string, scope: s
 
     configWithPath.content.compilerOptions ||= {};
     configWithPath.content.compilerOptions.paths ||= {};
-    configWithPath.content.compilerOptions.paths[`@${scope}/${projectName}`] = [
+    configWithPath.content.compilerOptions.paths[`${scope ? '@' + scope + '/' : ''}${projectName}`] = [
       `${relativeTargetPath}/src/index`
     ];
-    configWithPath.content.compilerOptions.paths[`@${scope}/${projectName}/*`] = [
+    configWithPath.content.compilerOptions.paths[`${scope ? '@' + scope + '/' : ''}${projectName}/*`] = [
```

Comment:
```suggestion
    configWithPath.content.compilerOptions.paths[`${scope ? `@${scope}/`: ''}${projectName}/*`] = [
```

Reply:
done in 
https://github.com/AmadeusITGroup/otter/pull/905